conf/cmi/block.block.ibmchatapp_pysakointi.yml

```diff
@@ -8,28 +8,30 @@ dependencies:
     - system
   theme:
     - hdbt_subtheme
-id: chatleijuke
+id: ibmchatapp_pysakointi
 theme: hdbt_subtheme
 region: attachments
-weight: -12
+weight: 0
 provider: null
-plugin: chat_leijuke
+plugin: ibm_chat_app
 settings:
-  id: chat_leijuke
-  label: 'Chat Leijuke Smartti'
+  id: ibm_chat_app
+  label: 'IBM Chat App (Pysäköinti)'
   label_display: '0'
   provider: helfi_platform_config
-  chat_title: 'Ask the robot about parking'
-  chat_selection: smartti_chatbot
+  hostname: 'https://coh-chat-app-test.mo1wrhhyog0.eu-de.codeengine.appdomain.cloud'
+  engagementId: parkki
+  tenantId: www-hel-fi-test
```

Comment:
These are incorrect parameters for the production site. I have asked for production parameters.